sr/comp/mixtape/magicq.py

```diff
@@ -1,32 +1,18 @@
-import socket
-import struct
-from typing import Tuple
+from typing import Union
 
+from pythonosc.udp_client import SimpleUDPClient  # type: ignore[import]
 
-class MagicqController:
-
-    def __init__(self, address: Tuple[str, int]) -> None:
-        self.address = address
-        self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
 
-    @staticmethod
-    def build_packet(data: bytes) -> bytes:
-        length = struct.pack('<H', len(data))
-        return b'CREP\0\0\0\0' + length + data
-
-    def send_command_once(self, command: str) -> None:
-        self.send_command(command, 1)
+class MagicqController:
 
-    def send_command(self, command: str, retries: int = 5) -> None:
-        packet = self.build_packet(command.encode('ascii'))
-        for _retry in range(retries):
-            self.socket.sendto(packet, self.address)
+    def __init__(self, host: str, port: int) -> None:
+        self.osc_client = SimpleUDPClient(host, port)
 
-    def activate_playback(self, num: int) -> None:
-        self.send_command(f'{num}A')
+    def activate_playback(self, playback: int) -> None:
+        self.osc_client.send_message(f'/pb/{playback}/go', None)
```

Comment:
The type signature of `send_message` doesn't allow a `None` second argument (though the implementation seems to). What's the expected behaviour here? (Should we be passing empty list `[]`?)

Reply:
This a mistake in typehints, the function has explicit handling of None.